marimo/_cli/file_path.py

```diff
@@ -126,24 +126,35 @@ def download(url: str) -> tuple[bool, str]:
         if url.startswith("https://static.marimo.app/"):
             return download(os.path.join(url, "download"))
```

Comment:
```suggestion
            return download(url)
```

static.marimo.app/e?slug=wxyz

not sure what the /download endpoint is